cmd/envs.go

```diff
@@ -88,15 +90,30 @@ var EnvsList = &cli.Command{
 }
 
 var EnvsNew = &cli.Command{
-	Name:      "new",
-	Usage:     "Create new environment",
-	UsageText: "nullstone envs new --name=<name> --stack=<stack> --provider=<provider>",
+	Name:        "new",
+	Description: "Creates a new environment in the given stack. The environment will be created as the last environment in the pipeline. Specify the provider, region, and zone to determine where infrastructure will be provisioned for this environment.",
+	Usage:       "Create new environment",
+	UsageText:   "nullstone envs new --name=<name> --stack=<stack> --provider=<provider>",
 	Flags: []cli.Flag{
-		&cli.StringFlag{Name: "name", Required: true},
+		&cli.StringFlag{
+			Name:     "name",
+			Usage:    "Provide a name for this new environment",
+			Required: true,
+		},
 		StackFlag,
-		&cli.StringFlag{Name: "provider", Required: true},
-		&cli.StringFlag{Name: "region"},
-		&cli.StringFlag{Name: "zone"},
+		&cli.StringFlag{
+			Name:     "provider",
+			Usage:    "Provide the name of the provider to use for this environment",
```

Comment:
"Provide ... the provider" reads awkward
```suggestion
			Usage:    "Set the name of the provider to use for this environment",
```

Reply:
I changed this to "Select the name of the provider to use for this environment"